app/scripts/lib/stores/extension-store.test.ts

```diff
@@ -89,7 +89,7 @@ describe('ExtensionStore', () => {
       const result = await localStore.get();
 
       expect(result).toBe(MOCK_STATE);
-      expect(getMock).toHaveBeenCalledWith(null);
+      expect(getMock).toHaveBeenCalledWith(['data', 'meta']);
```

Comment:
This test changed because we now only `get` the properties we need from the database, not any and everything.